multi_partner_learning.py

```diff
@@ -487,6 +528,38 @@ def init_with_agg_models(self):
         for partner in self.partners_list:
             partners_model_list.append(self.build_model_from_weights(self.aggregate_model_weights()))
         return partners_model_list
+    
+    
+    def init_with_previous_learned_model(self):
+        """Return a new model aggregating models from model_list"""
+        
+        previous_model = self.generate_new_model()
+        previous_model.load_weights(self.init_model_from)
+        previous_model.compile(
+            loss=keras.losses.categorical_crossentropy,
+            optimizer="adam",
+            metrics=["accuracy"],
+        )
+
+        return previous_model
```

Comment:
Hmm, this one is really close to `build_model_from_weights()`. Wouldn't it make sense to have only one of those two? For that, attribute `init_model_from` could be considered as and renamed to `weights_from_a_previous_model`. What do you think @celinejacques ?

Reply:
I'll have a look on it.